atomic_reactor/utils/operator.py

```diff
@@ -552,6 +556,16 @@ def __init__(self, files):
         """
         self.files = files
 
+    @property
+    def csv(self):
+        """
+        :return: ClusteredServiceVersion object of the operator manifests
+        :rtype: OperatorCSV
+        """
+        if len(self.files) > 1:
+            raise ValueError("Multiple CSV files found, only one CSV file is allowed")
```

Comment:
since you are raising here already again you can remove the same check from : https://github.com/containerbuildsystem/atomic-reactor/pull/1541/files

Reply:
Can we leave it as is for now? I don't want to include even more changes to this PR